lib/js/app/queryCreator/App.tsx

```diff
@@ -70,52 +52,42 @@ const App: FC<Props> = () => {
     <div>
       <QueryArguments />
       {showField('filters', analysis) && (
-        <FiltersSettings>
-          <Card>
-            <Title isDisabled={!collection}>{text.filters}</Title>
-            {isSchemaExist && !isSchemaLoading && (
-              <Filters
-                collection={collection}
-                filters={filters}
-                onReset={() => dispatch(setFilters([]))}
-                onRemove={(idx) => dispatch(removeFilter(idx))}
-                onChange={(idx, filter) => dispatch(updateFilter(idx, filter))}
-              />
+        <Card>
+          <Filters
+            collection={collection}
+            filters={filters}
+            onReset={() => dispatch(setFilters([]))}
+            onRemove={(idx: number) => dispatch(removeFilter(idx))}
+            onChange={(idx: number, filter: Filter) =>
+              dispatch(updateFilter(idx, filter))
+            }
+            onClick={(idx: string) => dispatch(addFilter(idx))}
```

Comment:
please unify `idx` as `string` or `number` if the same `idx` is returned from the above functions